toolkits/springer/packages/springer-media/scss/50-components/_media.scss

```diff
@@ -67,6 +67,7 @@
 
 .c-media__title {
 	margin-bottom: $media--title-bottom-spacing;
+	font-size: $context--font-size-lg;
```

Comment:
In the readme, the html example specified using `u-text-lg`. We don't want to depend on utility classes in components as they should be self-contained. We are able to specify this within the component CSS without using the utility class. Shown here.

The utility class was also not used in the demo, which means the final rendered component looks different (and may well look different in production if the utility class was not added). See example below

![example](https://user-images.githubusercontent.com/853933/144042685-69fd0838-4c08-4b90-a3ac-1194070e2cd1.png)